docs/source/image_classification.mdx

```diff
@@ -75,6 +75,9 @@ You can verify the transformation worked by indexing into the `pixel_values` of
 >>> plt.imshow(img)
 ```
 
+To learn how to leverage 🤗 datasets end-to-end for training an image classification model, refer to
```

Comment:
I think these would look nicer in the docs with a `<Tip>` component (it's a custom thing we have for the Hub docs). See https://github.com/huggingface/transformers/blob/main/docs/source/en/tasks/image_classification.mdx?plain=1#L175-L179 as an example

Reply:
Done.